packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from "react";
+
+import FieldHeading from "./field-heading.js";
+import TextField from "./text-field.js";
+
+export default class LabeledTextField extends React.Component<{||}> {
```

Comment:
NIT: Since we almost always will end up adding a prop some day, even when props are empty could you define the type:

`type Props = {||};`

It keeps things consistent which helps folks read the code and it simplifies maintenance when we have to start adding props.